index.rst

```diff
@@ -4,9 +4,184 @@ File Formats and Layouts for Cell-based Coadds
+  This isn't something third-party FITS compress-while-writing libraries can do, but if we're doing our own quantization anyway, it'd be straightforward to include.
+  Compression would not be compatible with storing the overlap regions in a binary table, but we would have the freedom to compress more than one cell at a time (and even compress more cells at a time in the overlap regions than in the inner regions).
+
+- Access to the outer cells on read would require more complex code and a few more seeks: first read the inner region, then read the four overlap regions (though these can almost certainly be read in pairs), and then put them all together.
+  This is not something we'd expect third-party general-purpose readers to ever do, but it's not a terribly complicated specification or a huge burden for, say, someone who wanted to write a Rubin-specific reader in a language other than Python.
+
+This option is also neutral to the problem of compressed subimage reads against object stores.
+
+Hybrid Options
+--------------
+
+We can in theory use a different approach for each image plane, though for the most part the arguments are the same for all image planes.
+The PSF images are the big exception: they are already intrinsically in their own coordinate system that doesn't have a meaningful WCS, and they are unlikely to ever be lossy compressed (I actually don't have any intuition for whether they'd have good lossless compression ratios).
+
+This makes data cube or exploded storage of PSFs quite attractive (binary tables too, at least if if we determine that we don't need to compress them at all), even if other image planes are stored in other ways.
+
+Metadata Layouts
 ================
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+Cell coadd metadata falls into three categories:
+
+- global information common to all cells (identifiers, WCS, grid structure);
+- per-cell information with a fixed schema (including coadded aperture corrections and wavelength-dependent throughput);
+- visit, detector and other IDs for the observations that contribute to each cell.
+
+While some global information will go into FITS headers (certainly the WCS and some identifiers), we do not want to assume that all global metadata can be neatly represented in the limited confines of a FITS header.
+A single-row binary table is another option, but we will likely instead adopt the approach recently proposed for other Rubin image data products on RFC-1030: embedding a JSON document as a byte array in a FITS extension HDU.
+
+A binary table with per-cell rows is a natural fit for the fixed-schema per-cell information, especially if the image data layout already involves a binary table with per-cell rows.
```

Comment:
I would encourage storing the fixed-schema data as a table even if it is also in the JSON.